gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -362,7 +359,7 @@ private int getSelectedProjectIndex(Project activeProject) {
 
   private void onShowAddFeatureDialogRequest(Point location) {
     if (!Loadable.getValue(viewModel.getActiveProject()).isPresent()) {
-      Log.e(TAG, "Attempting to add feature while no project loaded");
+      Timber.e("Attempting to add feature while no project loaded");
```

Comment:
Still seeing a few of these Timber changes but it's super picky so LGTM.

Reply:
I just thought of cleaning it up while we were editing the file. But will remember the suggestion for future PRs. Thanks

Reply:
+1 to cleaning these up in bulk in a separate PR, but ok for this one.